.github/labeler.yml

```diff
@@ -27,6 +35,7 @@ page:
 
 gallery:
 - src/components/GalleryPage/**/*
+- src/components/HomePage/Gallery.js
 - src/pages/gallery.js
```

Comment:
Also add `src/data/winners/*`. This is a new file Anakin is adding in his PR (#215)  (it will exist once we merge it it).

We'll also eventually remove `src/pages/gallery.js` (it is removed in Anakin's PR too) but for now we can leave it until Anakin's PR is merged